### PR TITLE
lindsay tries some css changes for code chunk style

### PR DESCRIPTION
**Pull request by lindsaycarr**
This reflects R style much better than before. Only thing I cannot figure out is why some of the output lines are considered class `hljs-title` and some are `hljs-comment`. I'd like them to all be styled like comments, but can't just change the title color because that class corresponds to other spots that should be colored black. Would be happy to chat about this in person @mwernimont . Also, if you have a better color than #aa5146 that passes the 508 contrast rules please let me know :)

Example of code output styled as title and then below it, code output styled as comments.

image

### CODE REVIEW

File: themes/hugo_theme_robust/static/css/styles.css [hunk 1]
```diff
@@ -40,17 +40,24 @@ caption {
...
+.hljs-title,
+.hljs-tag,
+.hljs-keyword,
+.hljs-variable,
+.hljs-attribute,
+.hljs-attr_selector,
 .hljs-built_in,
 .hljs-preprocessor{
-  color:#000;
+  color:#000  !important;;
```
Comment (jiwalker-usgs) [inline themes/hugo_theme_robust/static/css/styles.css]: double semicolon probably not what you want here.

File: themes/hugo_theme_robust/static/css/styles.css [hunk 2]
```diff
@@ -40,17 +40,24 @@ caption {
   line-height:1.5em;
 }
 
-.hljs-string{
-  color:#183691;
+.hljs-number{
+  color:#aa5146  !important;
```
Comment (jiwalker-usgs) [inline themes/hugo_theme_robust/static/css/styles.css]: It is usually a sign that something is wrong if you need to add `!important` in css.  Do you know what might be at issue here?

Comment (lindsaycarr) [reply themes/hugo_theme_robust/static/css/styles.css]: there's some other css built into Hugo somewhere that is tacked on at the end (or so it seems). Something called `solarized_dark.min.css`

File: themes/hugo_theme_robust/static/css/styles.css (hunk 1 above)
Comment (lindsaycarr) [reply themes/hugo_theme_robust/static/css/styles.css]: Thanks for catching that! Unfortunately made no difference to the title vs comment issue.

**Comment by mwernimont** [comment]
@lindsaycarr Does Hugo have some sort of documentation? Are we just missing something that is making Hugo think our intended comments are supposed to be titles? Could be a simple HTML element that Hugo triggers to be titles or something simple like that.

**Comment by lindsaycarr** [comment]
They do have documentation - https://gohugo.io/documentation/ . I looked in the markdown and compared the lines that were treated as titles instead of comments to the ones correctly treated as comments. I didn't notice anything obviously different about them, but maybe you might?

**Comment by lindsaycarr** [comment]
@mwernimont I did it (but had to use `!important` again.....)

**Comment by mwernimont** [comment]
Hmmm....I'll play around with that a bit this morning, I think its just something we may have to do, but always worth trying to avoid.

**Comment by mwernimont** [comment]
So was able to get without !important's, just gonna merge this and make a PR with the fix.

**Pull request merged**